tests/perf-test/OptionsListUtils.perf-test.ts

```diff
@@ -105,9 +105,12 @@ describe('OptionsListUtils', () => {
     });
 
     /* Testing getFilteredOptions */
-    test('[OptionsListUtils] getFilteredOptions with search value', async () => {
```

Comment:
We're still passing search value, then why this change?

Reply:
Oh, initially I removed the search value from this test to make it consistent with the use of the function now but then changed because this is a performance test. So, I forgot to update the test name back. Will update.